images/linux/scripts/helpers/install.sh

```diff
@@ -95,3 +95,68 @@ get_github_package_download_url() {
     fi
     echo $downloadUrl
 }
+
+get_github_package_hash() {
+    local repo_owner=$1
+    local repo_name=$2
+    local file_name=$3
+    local url=$4
+    local version=${5:-"latest"}
+    local prerelease=${6:-false}
+    local delimiter=${8:-'|'}
+    local word_number=${9:-2}
+
+    if [ -n "$url" ]; then
+        release_url="$url"
+    else
+        if [ "$version" == "latest" ]; then
+            release_url="https://api.github.com/repos/${repo_owner}/${repo_name}/releases/latest"
+        else
+            json=$(curl -s "https://api.github.com/repos/${repo_owner}/${repo_name}/releases?per_page=100")
+            tags=$(echo "$json" | jq -r --arg prerelease "$prerelease" '.[] | select(.prerelease == ($prerelease | test("true"; "i"))) | .tag_name')
+            tag=$(echo "$tags" | grep -o "$version")
```

Comment:
What if several tags pass this filter? E.g. when version specifies only major and minor part but there are multiple patches available.

Reply:
Well, in general, a mechanism with a version is just for specifying the exact value. If someone indicates a range of versions, then of course they will do it in vain.
But sounds reasonable to save future us from such a mistake. I'll add something here.